src/main/java/ui/components/pickers/MilestonePickerDialog.java

```diff
@@ -280,6 +294,12 @@ private HBox createPreviouslyAssignedMilestoneBox() {
                 .findAny();
     }
 
+    private Optional<PickerMilestone> getNextOpenMilestone(List<PickerMilestone> milestoneList) {
```

Comment:
Should move `getExistingMilestone` to be a `static` method in `PickerMilestone` class as well. (Sorry for the extra work)